ts/Dashboards/Components/DataGridComponent/DataGridComponent.ts

```diff
@@ -289,9 +289,10 @@ class DataGridComponent extends Component {
      * @internal
      */
     public override getOptions(): Partial<Options> {
+        const optionsCopy = merge(this.options);
+        optionsCopy.gridOptions = this.grid?.getOptions(true);
```

Comment:
It's true by default
```suggestion
        optionsCopy.gridOptions = this.grid?.getOptions();
```